2.0/m_getbans.cpp

```diff
@@ -0,0 +1,65 @@
+/*
+ * InspIRCd -- Internet Relay Chat Daemon
+ *
+ *   Copyright (C) 2014 Miguel Peláez <miguel2706@outlook.com>
+ *   Copyright (C) 2014-2015 Hira.io Team <staff@hira.io>
+ *
+ * This file is part of InspIRCd.  InspIRCd is free software: you can
+ * redistribute it and/or modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation, version 2.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+/* $ModAuthor: Hira.io Team */
+/* $ModDesc: Get the bans from other channels. Usage: /ban j:#channel*/
+/* $ModDepends: core 2.0 */
+
+#include "inspircd.h"
+class ModuleGetBans : public Module
+{
+ public:
+	void init()
+	{
+		ServerInstance->Modules->Attach(I_OnCheckBan, this);
```

Comment:
You could use an `Implementation[]` here so that if anyone wants to extend the module in the future they can do so easily.